core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AnalysisUtils.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.tool.analysis
+
+object DiagnosticMetrics {
```

Comment:
I think this was mentioned in another comment, the name here seems to generic.  What is it Diagnostics for? In this case I think its Stage/Task level metrics, correct?  So if aggregating at stage level then perhaps put Stage in the name of it.

Reply:
thanks! changed this to `StageAccumDiagnosticMetrics` to show it is stage-level and comes from `Accumulable`.